alembic/versions/a9227b0cb10b_move_real_paths_to_prompt.py

```diff
@@ -0,0 +1,44 @@
+"""move real paths to prompt
+
+Revision ID: a9227b0cb10b
+Revises: 4f05e794f2b2
+Create Date: 2025-10-13 22:09:33.896758
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "a9227b0cb10b"
+down_revision: Union[str, None] = "4f05e794f2b2"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "validator_requests",
+        sa.Column("real_prices", sa.JSON(), nullable=True),
+    )
+    # Migrate data from miner_scores to validator_requests
+    op.execute(
+        """
+        UPDATE validator_requests vr
```

Comment:
you've probably double-checked it already, but just in case:
we're updating a single row in the validator_requests table, but in miner_scores we have many identical real_prices for that single validator request, I just wonder if it still works correct, I would think of something like taking only the very first record from miner_scores for real_prices

but if everything is fine with it, then it doesn't matter